README.md

```diff
@@ -173,7 +173,7 @@ type Context struct {
 	EventID string
 	
 	// Checksum of the EdgeX Event -- will be filled for a received CBOR Event
-	EventChecksum string
+	EventChecksum string 
```

Comment:
Do we need this change with extra space?

Reply:
not needed. just removed it